src/main/kotlin/cryptoservice/service/EncryptionService.kt

```diff
@@ -21,8 +22,13 @@ class EncryptionService {
         riScId: String,
     ): String {
         return try {
+            val tempFile = File.createTempFile("sopsConfig-$riScId-${System.currentTimeMillis()}", ".yaml")
+            println("Temporary file created here: ${tempFile.absolutePath}")
+            tempFile.writeText(config)
+            tempFile.deleteOnExit()
+
```

Comment:
Hvorfor gjør vi dette? 🙈

Reply:
Fordi man er nødt til å ha en config-fil slik det er i dag - skal fjerne println, den overså jeg visst 😊
I den tidligere varianten av sops så sendte vi med et flagg for å sende config som json. For å forenkle sops så masse som mulig så la jeg kun med det vi ikke har mulighet til å provide på noen som helst annen måte (kun gcp access token).

Fordi .sops.yaml allerede lagres i github og det ikke ligger noen skummel informasjon i de filene, så laget jeg temp-filer for disse som slettes etter at de har blitt brukt. Tenkte at tiden man bruker på å lage en tmp-fil vil være utrolig liten ift tiden man bruker på kryptering ✨

Hvis man skal se på hva man heller burde gjort, så ville nok den beste løsningen være å  gjøre sops edit istedenfor. Og på den måten vil man ha muligheten til å bruke configen som ligger i metadata-biten av en kryptert fil.

Reply:
I/O er ofte mykje meir kostbare operasjoner enn CPU. Men eg tviler ikkje på at dette sikkert er heilt innafor tidsmessig 😊